plugin/entryBase.go

```diff
@@ -3,7 +3,13 @@ package plugin
 import (
 	"context"
 	"flag"
+	"fmt"
+	"reflect"
 	"time"
+
+	"github.com/ekinanp/jsonschema"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
We should definitely avoid pulling k8s in as a dependency of the plugin module. That's going to cause real problems if we eventually split individual plugins out as separate artifacts to be more modular.